pkg/controller/jobframework/webhook/defaulter_test.go

```diff
@@ -36,7 +38,20 @@ var (
 )
 
 type TestResource struct {
```

Comment:
Let's also add an `inline` field, just to make sure we are being comprehensive enough.

Reply:
Done